src/test/java/tech/jhipster/lite/module/domain/resource/JHipsterModulesResourcesTest.java

```diff
@@ -39,4 +39,11 @@ void shouldNotBuildWithDuplicatedSlug() {
       DuplicatedSlugException.class
     );
   }
+
+  @Test
+  void shouldHaveMeaningfullToString() {
+    var resource = defaultModuleResourceBuilder().slug("dummy").build();
+
+    assertThat(resource.toString()).contains("JHipsterModuleResource", "slug=", "dummy");
```

Comment:
With SHORT_PREFIX_STYLE, you can then test the full toString expression:
```suggestion
    assertThat(resource).hasToString("...");
```

Reply:
Even with the SHORT_PREFIX_STYLE, the result is too verbose, that I would rather not put the whole String in the test